git-worktree/src/lib.rs

```diff
@@ -1 +1,111 @@
 #![forbid(unsafe_code, rust_2018_idioms)]
+//! Git Worktree
+
+use git_hash::oid;
+use git_index::{
```

Comment:
Let's fully qualify the path to `git_index` items, instead of importing them to somewhat generic type names.